src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -3,11 +3,17 @@ setlocal
 
 echo BridgeKeepRunning=%BridgeKeepRunning%
 if '%BridgeKeepRunning%' neq 'true' (
-    echo Stopping the Bridge...
+    echo Releasing Bridge resources and stopping it if running locally.
     pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    call Bridge.exe -stopIfLocal %*
+    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
+    call Bridge.exe -stopIfLocal -reset %*
     popd
 ) else (
+    echo Releasing Bridge resources but leaving it running.
+    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+    echo Invoking Bridge.exe -reset %* ...
+    call Bridge.exe -reset %*
+    popd
     echo The Bridge was left running because BridgeKeepRunning is true
```

Comment:
CleanupWcfTestService.cmd is the script called to clean up all resources -- that's its purpose.  In this case, cleanup means remove all certs and firewall rules created, shutdown the AppDomains, and continue running.  The Bridge will reacquire resources the next time it receives requests.

I think it is fairly critical that after a run all DLL's loaded from the BridgeResourceFolder be releaesd, otherwise the build will fail because it cannot write the new DLL's into the BridgeResourceFolder.